blog/_posts/2025-04-22-Enhance-security-OS-Security-Patch.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Enhancing Your Operating System's Security with OS Security Patches in AKS"
+description: "Learn the what, the whys, and the hows of configuring OS Security Patch Auto upgrade channel.  The article also covers some reasons for OS Security Patch, with some primary benefits being increased performance, better security, and minimal disruption to workloads."
+date: 2025-04-22
+authors:
+- Kaarthikeyan Subramanian
+- Ye Wang
+categories: 
+- Security
+- Operations
+---
+
+
+## Why Do We Need Security Patches at the Node Operating System Level?
+
+Operating System (OS) security patches are crucial for maintaining the integrity and security of your systems. These patches address vulnerabilities that could be exploited by malicious actors, ensuring that your system remains protected against the latest threats. Regular OS security patches safeguard data, ensure compliance, and prevent disruptions. At Azure Kubernetes service we are constantly working hard and innovating to guarantee maximum security for your Node operating systems. 
+Presently there are two ways to apply security patches at the OS level automatically with AKS. Read more [OS Security Patch](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli)
+
+**Auto Node Image Channel** - AKS updates nodes weekly with a newly patched VHD for security and bug fixes. This update, which follows maintenance windows and surge settings, incurs no extra VHD cost. Linux unattended upgrades are disabled by default when using this channel. Node image upgrades are supported as long as the cluster's Kubernetes minor version is in support. These AKS-tested node images are fully managed and applied with safe deployment practices.
```

Comment:
Is this the name? I always refer to it as Automatic Node Image Upgrade channel. 
Even if cutting the upgrade, I'd suggest not shortening automatic.

Reply:
I don't think there should be any mention here of VHD cost, if you want to mention costs below on the next channel it's fine but here, given it's the first option it looks out of place. "VHD cost? why would it incur any VHD cost at all?"

I understand you're trying to compare it with the below but that information hasn't come up in the blog yet.

Reply:
Similarly, it's equally out of place to bring unattended upgrades here without any prior explanation, most people have no idea what unattended upgrades are. Also that's an Ubuntu thing, not a Linux thing, it has slightly different names for similar functionality in other OSs.

If you want to include it here, you need to do a preamble before where you touch upon it, and maybe describe classical ways in how patching is done / approached in VMs before.

Reply:
Removed VHD cost and put it under comparison table. Added a clear preamble on traditional sec patching and the gaps there before introing our two managed options.